wsapi.go

```diff
@@ -423,17 +448,11 @@ func (s *Session) RequestGuildMembers(guildID, query string, limit int) (err err
 		return ErrWSNotFound
 	}
 
-	data := requestGuildMembersData{
-		GuildID: guildID,
-		Query:   query,
-		Limit:   limit,
-	}
-
 	s.wsMutex.Lock()
 	err = s.wsConn.WriteJSON(requestGuildMembersOp{8, data})
 	s.wsMutex.Unlock()
 
-	return
+	return err
```

Comment:
This is unnecessary as `err` is assigned on line 452.

Reply:
It's required to return err as the function returns an error?

Reply:
Since `err` is a named return value, it's redundant. See: [named return values](https://tour.golang.org/basics/7)

Reply:
TIL. Thanks.